cms/models.py

```diff
@@ -244,10 +244,15 @@ class BootcampRunPage(BootcampPage):
     bootcamp_run = models.OneToOneField(
         "klasses.BootcampRun",
         null=True,
+        blank=True,
         on_delete=models.SET_NULL,
         help_text="The bootcamp run for this page",
     )
 
+    def copy(self, *args, **kwargs):  # pylint: disable=signature-differs
+        kwargs["exclude_fields"] = kwargs.get("exclude_fields", []) + ["bootcamp_run"]
```

Comment:
This works well, but copied bootcamp run pages should be unpublished by default since (a) they don't have a run associated with them when they're first copied, and (b) they still have text/images/etc. from an existing bootcamp run page. I tinkered with this a little and I couldn't find an easy way to validate these things before publishing, but we can pass in an extra kwarg here that will default the page to be unpublished. That will make it less likely for us to have a published bootcamp run page that isn't yet ready to be viewed.

```suggestion
        kwargs["exclude_fields"] = kwargs.get("exclude_fields", []) + ["bootcamp_run"]
        kwargs["keep_live"] = False
```

Reply:
Thanks @gsidebo for the detailed review. I have added the required change.

Just a question though: do we need to ask the `course authoring team` about the `Pages` they are usually copying so we can override their copy method as well to ensure that user experience never gets affected on prod, by this at least?

Reply:
@HamzaIbnFarooq We haven't had any issues yet with the existing workflow, so I think we can wait until an issue comes up before making any changes to other copy methods